src/ngx_http_lua_common.h

```diff
@@ -96,6 +96,10 @@ typedef struct {
 #define NGX_HTTP_LUA_MAX_ARGS 100
 #endif
 
+#ifndef NGX_HTTP_LUA_MAX_PHASE_COUNT
+#define NGX_HTTP_LUA_MAX_PHASE_COUNT 10
```

Comment:
I'm afraid this macro name can be confusing. We are not limiting the number of phases but rather the number of the handlers in *each* phase, right? Let's find a better name :)